scripts/update-authors.cjs

```diff
@@ -0,0 +1,27 @@
+const gitAuthors = require('grunt-git-authors')
```

Comment:
Think this dependency was removed

Reply:
Good catch. Was, and still am, in two minds about using it or just keeping the authors/collaborator list up to date manually.